### PR TITLE
netty: Remove RequestMessagesCommand

### PR DESCRIPTION
**Pull request by buchgr**
It's not needed. Can do the same by scheduling a task on the eventloop.

### CODE REVIEW

File: netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java [hunk 1]
```diff
@@ -1,52 +0,0 @@
...
-  private final int numMessages;
-  private final AbstractStream2.TransportState state;
-
-  public RequestMessagesCommand(AbstractStream2.TransportState state, int numMessages) {
-    this.state = state;
-    this.numMessages = numMessages;
-  }
-
-  void requestMessages() {
-    state.requestMessagesFromDeframer(numMessages);
```
Comment (ejona86) [inline netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java]: Can requestMessagesFromDeframer get its visibility reduced?

Comment (buchgr) [inline netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java]: No cause it sits in `AbstractStream2`.

Comment (ejona86) [inline netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java]: It looks like it could actually become `protected` in `AbstractStream2`

Comment (buchgr) [inline netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java]: I might be missing something, but `requestMessagesFromDeframer` is defined in class `io.grpc.internal.AbstractStream2.TransportState` and called in class `io.grpc.netty.NettyClientStream.Sink` ... I don't see how anything but `public` would work?

Comment (ejona86) [inline netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java]: Ah, yeah. Agreed.

**Pull request merged**